content/en-us/reference/engine/enums/AudioApiRollout.yaml

```diff
@@ -1,23 +1,25 @@
 name: AudioApiRollout
 type: enum
-summary: ''
-description: ''
+summary: |
+  Used to determine whether voice chat uses the Audio API
+description: |
+  Used to determine whether voice chat uses the Audio API
 code_samples: []
 tags: []
 deprecation_message: ''
 items:
   - name: Disabled
-    summary: ''
+    summary: 'Voice chat will not use the Audio API. Roblox will use an internal voice chat implementation that is automatic and hidden.'
     value: 0
     tags: []
     deprecation_message: ''
   - name: Automatic
-    summary: ''
+    summary: 'Currently means the same thing as Disabled, but will be updated to mean Enabled in the future. This is the default behavior.'
     value: 1
     tags: []
     deprecation_message: ''
   - name: Enabled
-    summary: ''
+    summary: 'Voice chat will use the Audio API, meaning it can be customized or controlled to sound different.'
```

Comment:
```suggestion
    summary: 'Voice chat will use the Audio API, via `Class.AudioDeviceInput`s, which can be customized or controlled to sound different.'
```

Reply:
Or maybe
```suggestion
    summary: 'Voice chat will use the Audio API, meaning it can be customized or controlled via `Class.AudioDeviceInput|AudioDeviceInputs`.'
```